ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh

```diff
@@ -151,7 +151,15 @@ function pr-prebuild-doxygen-tag(){
                             fi
 
                             # Check the doxygen tag written in upper case beacuase doxygen cannot use upper case tag such as '@TODO'.
-                            if [[ $line =~ "@"[A-Z] ]]; then
+			    # This is a simple (incorrect) method to detect.
+			    # @todo it should be able to know if it is inside a comment or not!
+                            if [[ $line =~ ^[\ \t]*\*[\t \t]*"@"[A-Z] ]]; then
```

Comment:
I think to check
```
/** @TAG
 * @TAG
```
This line should be cheanged
```suggestion
                            if [[ $line =~ [\ \t]*\*[\t \t]*"@"[A-Z] ]]; then
```